scripts/pi-hole/php/header.php

```diff
@@ -185,152 +74,43 @@ function getTemperature()
         html { background-color: #000; }
     </style>
 <?php } ?>
-    <link rel="stylesheet" href="style/vendor/SourceSansPro/SourceSansPro.css?v=<?php echo $cacheVer; ?>">
+
+    <!-- Common styles -->
     <link rel="stylesheet" href="style/vendor/bootstrap/css/bootstrap.min.css?v=<?php echo $cacheVer; ?>">
+<?php if ($auth) { ?>
     <link rel="stylesheet" href="style/vendor/datatables.min.css?v=<?php echo $cacheVer; ?>">
     <link rel="stylesheet" href="style/vendor/datatables_extensions.min.css?v=<?php echo $cacheVer; ?>">
     <link rel="stylesheet" href="style/vendor/daterangepicker.min.css?v=<?php echo $cacheVer; ?>">
+<?php } ?>
     <link rel="stylesheet" href="style/vendor/AdminLTE.min.css?v=<?php echo $cacheVer; ?>">
     <link rel="stylesheet" href="style/vendor/select2.min.css?v=<?php echo $cacheVer; ?>">
 
 <?php if (in_array($scriptname, array('groups.php', 'groups-adlists.php', 'groups-clients.php', 'groups-domains.php'))) { ?>
```

Comment:
Do you want to leave this here or move it to `header_authenticated.php`? It won't be used on the login page.

Reply:
Same as above.